src/app/(authorized)/dashboard/src/ui/calendar/days.tsx

```diff
@@ -0,0 +1,51 @@
+import { useMemo } from "react"
+
+import { getCalendarList } from "@/utils/calendar"
+import { Dayjs } from "dayjs"
+
+import { CalendarType } from "@/types/calendar"
+import { cn } from "@/lib/utils"
+
+const DAY_LIST = ["일", "월", "화", "수", "목", "금", "토"]
+
+interface DaysProps {
+  date: Dayjs
+}
+
+const Days = ({ date }: DaysProps) => {
+  const year = date.get("year")
+  const month = date.get("month") + 1
+
+  const dateList = useMemo(() => getCalendarList(year, month), [year, month])
+
+  return (
+    <div>
+      <div className="calendar-row">
+        {DAY_LIST.map((day) => (
+          <div className="calendar-cell justify-center">{day}</div>
+        ))}
+      </div>
+      {dateList.map((week, index) => (
+        <div className="calendar-row" key={`${year}-${month}-${index}`}>
+          {week.map(({ date, type }) => {
+            const dateString = String(date.get("date"))
+            const isCurrent = type === CalendarType.Current
+
+            return (
+              <div
+                key={`${year}-${month}-${index}-${date.get("d")}`}
+                className={cn("calendar-cell h-20", {
+                  "text-gray-400": !isCurrent,
+                })}
```

Comment:
P2: 현재 적용되어 있는 현재 달이 아닌 경우의 색상 분기보다 더 선명하게 적용되면 좋을 것 같습니다!

```suggestion
                className={cn("calendar-cell h-20", {
                  "text-gray-500": !isCurrent,
                })}
```

아래 이미지는 좌측이 적용 전, 우측이 `500` 적용 후 입니다!

<img width="213" alt="스크린샷 2024-01-22 오후 1 56 54" src="https://github.com/p-acid/easy-tax/assets/87939521/0712fbcd-fa3f-47c6-86b9-e644c3466144">
<img width="210" alt="스크린샷 2024-01-22 오후 1 57 02" src="https://github.com/p-acid/easy-tax/assets/87939521/9bc8ecc2-4d69-45c4-8c87-89af8481c913">